dbcrossbar/src/drivers/redshift/write_remote_data.rs

```diff
@@ -267,7 +267,8 @@ impl VerifyRedshiftCanImportFromCsv for DataType {
             | DataType::Json
             | DataType::Named(_) // We could expand these, maybe.
             | DataType::Struct(_)
-            | DataType::Uuid => Err(format_err!(
+            | DataType::Uuid
+            | DataType::TimeWithoutTimeZone => Err(format_err!(
```

Comment:
left out on purpose, as i don't have the means to test on redshift.

Reply:
Honestly, I test only rarely on RedShift, because it's expensive and obnoxious to spin up a cluster. This is one of the few drivers that doesn't attempt to round-trip `exact_output.csv`, because it's missing several fundamental data types. So it's totally fine to error out in this case.

So in this one case, we can punt. Well, this and Shopify, which will be deprecated in favor of JSON Lines support soon.